tests/virtctl/usbredir.go

```diff
@@ -161,23 +145,23 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 		It("Should work several times", func() {
 			for i := 0; i < 4*v1.UsbClientPassthroughMaxNumberOf; i++ {
 			retry_loop:
-				for try := 0; try < 3; try++ {
+				for try := range numTries {
 					ctx, cancelFn := context.WithCancel(context.Background())
 					errch := make(chan error)
-					go runConnectGoroutine(virtClient, name, namespace, ctx, errch)
+					go runConnectGoroutine(name, namespace, ctx, errch)
 
 					select {
 					case err := <-errch:
 						cancelFn()
-						time.Sleep(100 * time.Millisecond)
-						if try < 3 {
```

Comment:
Replace if/else with `Expect(try).To(BeNumerically("<", numTries))`?

Reply:
Sure.